### ISSUE TITLE
Unread tab on the new mobile layout only shows highlights

### ISSUE THREAD

**Issue by clokep** [vector-im/element-meta#649]
**Describe the bug**
The unread tab on the new mobile layout only shows rooms with unread notifications (i.e. highlight messages).

If you have notifications disabled for most rooms than this filter becomes useless and isn't an easy way to track unread rooms.

**To Reproduce**

1. Disable notifications most notifications, e.g. something like:

![image](https://user-images.githubusercontent.com/517124/191538494-5a513983-26a3-4892-ac19-273b6374a129.png)

2. Have rooms with unread messages (that aren't highlights).

**Expected behavior**
Any room with an unread message (i.e. my read receipt is not on the last event in the room) would be shown on the unread tab.

**Screenshots**

| All chats | Unread |
|---|---|
| ![image](https://user-images.githubusercontent.com/517124/191543614-0d9f9ba7-194e-40d4-bb85-1889af7085a4.png) |![image](https://user-images.githubusercontent.com/517124/191543678-c28975a6-464b-4e64-90fd-376ce3fe53f0.png) |

Note how synapse admins doesn't show up on the unread tab although I have unread messages in it (black line on the left shows it is unread).

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6] iPhone 14
 - OS: [e.g. iOS8.1] iOS 16.0.1
 - Version [e.g. 22] 1.9.6 (20220920134957)

CC @daniellekirkwood

**Comment by SimonBrandner** [comment]
For me, the current behavior is exactly what I want - the rooms where I've disabled notifications aren't that important to me, so I don't want them to show up on that list

**Comment by clokep** [comment]
> For me, the current behavior is exactly what I want - the rooms where I've disabled notifications aren't that important to me, so I don't want them to show up on that list

This sounds more like you want the "Notifications" view from Element Web which only shows notifications then?

To be clear by "disabled notifications" I mean having the notifications setting to "off" in the settings panel (this causes the room to get a dot on it when new messages come in), which isn't quite the same as setting notifications to "None" in a specific room's drop down (which causes no change in status when new messages come in).

(Why is the terminology here so confusing? 😢 )

**Comment by SimonBrandner** [comment]
> > For me, the current behavior is exactly what I want - the rooms where I've disabled notifications aren't that important to me, so I don't want them to show up on that list
> 
> This sounds more like you want the "Notifications" view from Element Web which only shows notifications then?

Yeah, basically that. Though I guess `Unreads` is more understandable for a regular user and that is why it was chosen

> To be clear by "disabled notifications" I mean having the notifications setting to "off" in the settings panel (this causes the room to get a dot on it when new messages come in), which isn't quite the same as setting notifications to "None" in a specific room's drop down (which causes no change in status when new messages come in).
> 
> (Why is the terminology here so confusing? cry )

To be clear about my side: I have a lot of room set to `Mentions & Keywords` but do not want them to show up on that list because I don't care too much about those rooms (unless I am mentioned/keyworded (if that's a word))

**Comment by clokep** [comment]
> To be clear about my side: I have a lot of room set to `Mentions & Keywords` but do not want them to show up on that list because I don't care too much about those rooms (unless I am mentioned/keyworded (if that's a word))

Sounds like I have the same setup but *do* care about those rooms. Rooms I don't care about I disable notifications completely.

**Comment by SimonBrandner** [comment]
> > To be clear about my side: I have a lot of room set to `Mentions & Keywords` but do not want them to show up on that list because I don't care too much about those rooms (unless I am mentioned/keyworded (if that's a word))
> 
> Sounds like I have the same setup but _do_ care about those rooms. Rooms I don't care about I disable notifications completely.

Yeah, I just only care about them if I get mentioned/keyworded

**Comment by daniellekirkwood** [comment]
There's an opportunity to revisit these decisions and this behaviour with EX. 
For me personally the Unread tab behaves as expected however, we should research whether there are others that feel similarly to @clokep here and expect more from the Unread tab. 

I believe that @amshakal & @VolkerJunginger were planning on changing the Filters a little, along with Cell Decoration (and the wider Notifications work) hopefully, even if we don't change the behaviour, the app still offers the view you're looking for...
